internal/util/projutil/project_util.go

```diff
@@ -143,7 +143,12 @@ func RewriteFileContents(filename, target, newContent string) error {
 		return err
 	}
 
-	err = ioutil.WriteFile(filename, []byte(modifiedContent), FileMode)
+	return WriteFile(filename, modifiedContent)
+}
+
+// WriteFile writes contents to file.
+func WriteFile(filename, content string) error {
```

Comment:
Nit: this seems like a superfluously shallow abstraction. Why is it so hard to call `ioutil.WriteFile()`?